### PR TITLE
Support publish the windows binary package and drawin binary package

### PR DESCRIPTION
**Pull request by zymap**
---
Issue: #122 

*Motivation*

Rlease the other OS binatry

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -17,8 +17,9 @@ jobs:
       - name: build
         run: |
           version=`cat VERSION`
-          go build -ldflags "-X github.com/streamnative/pulsarctl/pkg/pulsar.ReleaseVersion=Pulsarctl-Go-$version" .
-
+          go build -o pulsarctl.sh -ldflags "-X github.com/streamnative/pulsarctl/pkg/pulsar.ReleaseVersion=Pulsarctl-Go-$version" .
```
Comment (sijie) [inline .github/workflows/release.yml]: why the output is `pulsarctl.sh`?

Comment (zymap) [reply .github/workflows/release.yml]: because it is conflict with the output of Darwin. I need a different name.

Comment (zymap) [reply .github/workflows/release.yml]: Maybe pulsarctl-linux is more better?

File: install.sh [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+# * Unless required by applicable law or agreed to in writing, software
+# * distributed under the License is distributed on an "AS IS" BASIS,
+# * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# * See the License for the specific language governing permissions and
+# * limitations under the License.
+# */
+
+set -e
+
+version=v0.1.0
```
Comment (sijie) [inline install.sh]: Can we create a `stable.txt` file to store the stable version? The installer script can read the stable version from this file.

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -17,8 +17,9 @@ jobs:
       - name: build
         run: |
           version=`cat VERSION`
-          go build -ldflags "-X github.com/streamnative/pulsarctl/pkg/pulsar.ReleaseVersion=Pulsarctl-Go-$version" .
-
+          go build -o pulsarctl-linux -ldflags "-X github.com/streamnative/pulsarctl/pkg/pulsar.ReleaseVersion=Pulsarctl-Go-$version" .
```
Comment (sijie) [inline .github/workflows/release.yml]: there are two variables on building `pulsarctl` - OS and ARCH. so we should use a consistent pattern for naming the built binary - `pulsarctl-<arch>-<os>`. 

For arch, please build both 386 and amd64.

File: README.md [hunk 4]
```diff
@@ -33,19 +33,38 @@ A CLI tool for the [Apache Pulsar](https://pulsar.incubator.apache.org/) project
...
+
+```bash
+sh -c "$(curl -fsSL https://raw.githubusercontent.com/streamnative/pulsarctl/master/install.sh)"
+```
 
-## Usage
+#### Install pulsarctl binary on Windows
+
+``` bash
+curl -LO https://github.com/streamnative/pulsarctl/releases/download/v0.1.0/pulsarctl.exe
```
Comment (sijie) [inline README.md]: Why can't install.sh handle installing pulsarctl binary on Windows?

File: install.sh [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
```
Comment (sijie) [inline install.sh]: A better way to implement the installer.

```
discoverArch() {
  ARCH=$(uname -m)
  case $ARCH in
    x86) ARCH="386";;
    x86_64) ARCH="amd64";;
    i686) ARCH="386";;
    i386) ARCH="386";;
  esac
}

# discovers the operating system for this system.
discoverOS() {
  OS=$(echo `uname`|tr '[:upper:]' '[:lower:]')

  case "$OS" in
    # Minimalist GNU for Windows
    mingw*) OS='windows';;
  esac
}

discoverArch
discoverOS

curl -# -LO https://github.com/streamnative/pulsarctl/releases/download/$version/pulsarctl-${ARCH}-${OS}
mv pulsarctl-${ARCH}-${OS} pulsarctl
chmod +x pulsarctl
mv pulsarctl /usr/local/bin
```

Comment (sijie) [reply install.sh]: Also please add `.exe` if OS is windows.

File: README.md [hunk 6]
```diff
@@ -50,10 +50,20 @@ sh -c "$(curl -fsSL https://raw.githubusercontent.com/streamnative/pulsarctl/mas
...
+If you have command-line shell tools, such as PowerShell. You can download pulsarctl by this way:
+
+```bash
+sh -c "$(curl -fsSL https://raw.githubusercontent.com/streamnative/pulsarctl/master/install.sh)"
+```
+
+Or you can download the executable directly.
+
 ``` bash
 curl -LO https://github.com/streamnative/pulsarctl/releases/download/v0.1.0/pulsarctl.exe
```
Comment (sijie) [inline README.md]: don't you need to fix this? we don't ship `pulsarctl.exe`.

File: install.sh [hunk 7]
```diff
@@ -20,15 +20,36 @@
...
+        SUFFIX='.exe'
+        ;;
+  esac
+}
+
+discoverArch
+discoverOS
+
+curl -# -LO https://github.com/streamnative/pulsarctl/releases/download/$version/pulsarctl-${ARCH}-${OS}${SUFFIX}
+mv pulsarctl-${ARCH}-${OS}${SUFFIX} pulsarctl
```
Comment (sijie) [inline install.sh]: do we need to keep the `${SUFFIX}`?

File: README.md (hunk 6 above)
Comment (zymap) [reply README.md]: Ah. Sorry about that. I need. Will fix it.

File: README.md [hunk 8]
```diff
@@ -36,33 +36,17 @@ A CLI tool for the [Apache Pulsar](https://pulsar.incubator.apache.org/) project
...
-Or you can download the executable directly.
-
-``` bash
-curl -LO https://github.com/streamnative/pulsarctl/releases/download/v0.1.0/pulsarctl-{ARCH}-windows.exe
-```
-
-To find out the latest stable version, take a look at https://raw.githubusercontent.com/streamnative/pulsarctl/master/stable.txt
+1. Download the executable from https://github.com/streamnative/pulsarctl/releases. 
+2. Add the pulsarctl directory to your system PATH.
+3. Input `pulsarctl -h` at CMD to test pulsarctl is work.
```
Comment (sijie) [inline README.md]: "Input" => "Type"

**Pull request merged**